app/services/spree_adyen/payment_sessions/find_or_create.rb

```
@@ -1,40 +1,50 @@
         @order = order
         @amount = amount
         @user = user
         @payment_method = payment_method
+        @channel = channel || SpreeAdyen::PaymentSession::AVAILABLE_CHANNELS[:web]
+        @return_url = return_url
       end
 
       def call
-        return unless order.can_create_adyen_payment_session?
-        return payment_session if payment_session.present?
+        return failure(nil, "Cannot create Adyen payment session for the order in the #{order.state} state") unless order.can_create_adyen_payment_session?
```

RuboCop (1.81.7)

[convention] Modifier form of `unless` makes the line too long.

(Style/IfUnlessModifier)